src/pages/Main/Ranking.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import RankList from "./RankList";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faChevronCircleLeft,
+  faChevronCircleRight,
+} from "@fortawesome/free-solid-svg-icons";
+import "./Main.scss";
+
+class Ranking extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      currentIndex: 0,
+    };
+  }
+
+  handleButtonClick = () => {
+    const { currentIndex } = this.state;
+    const imgWidth = 20;
+    const imgViewLength = 5;
+    const imgTranslate = imgWidth * imgViewLength;
+    if (currentIndex === 0) {
+      this.setState({
+        currentIndex: currentIndex - imgTranslate / 2,
+      });
+    } else {
+      this.setState({
+        currentIndex: currentIndex + imgTranslate / 2,
+      });
+    }
+  };
+
+  render() {
+    const { currentIndex } = this.state;
+    return (
+      <div className="ranking">
+        <div className="ranking-title">{this.props.title}</div>
+        <div className="ranking-carousel">
+          <ul style={{ transform: `translateX(${currentIndex}%)` }}>
+            {this.props.ranking.map((rank) => {
+              return (
+                <RankList
+                  key={rank.id}
+                  image={rank.image}
+                  name={rank.cafeName}
+                  address={rank.address}
+                  star={rank.star}
+                />
+              );
+            })}
+          </ul>
+          <div
+            className={
+              "carousel-btn left-btn " + (!currentIndex ? "btn-remove" : "")
+            }
```

Comment:
해당 부분 템플릿 리터럴을 사용하는 방법으로 변경해주세요!
```suggestion
            className={`carousel-btn left-btn ${!currentIndex && "btn-remove"}`}
```